lambda-code/submission/main.ts

```diff
@@ -59,7 +59,7 @@ export const handler: Handler = async (submission: AnyObject) => {
       })
     );
 
-    return { status: false };
+    return { status: false, submissionId };
```

Comment:
We probably don't need to return a `submissionId` if there is a failure because that identifier won't mean anything.

Reply:
☝️ what he said....
Ignore my comment below.

Reply:
ya we can remove was debating about keeping the return signatures the same for both --- I know with the current we don't need it.

Reply:
updated